### ISSUE TITLE
Create an only "Statistics User" user role 

### ISSUE THREAD

**Issue by Chinedar** [SORMAS-Foundation/SORMAS-Project#3466]
### Situation Description
Currently all web user roles have access to the statistics module and other modules
### Feature Description
Create a user role that only allows the "statistics user" access to the statistics module (without access to the database export menu within the statistics module. 
### Additional Information
This will enable access to be given to partners and stakeholders who can carry out basic analytics without accessing the main database with PIIs.

**Comment by bernardsilenou** [comment]
@Chinedar Only admins have access to the database export, right?
- What are the rights that this user should have?
- Should they see only the statistics menu and nothing else?

**Comment by Chinedar** [comment]
@bernardsilenou No, national users also have access to database export. 
- right to manipulate the statistics menu as it is
- They should only see the statistics and dashboard menu

**Comment by Chinedar** [comment]
@bernardsilenou if we restrict to a particular disease when creating this user, does it restrict the statistics menu to only data from the selected disease?

**Comment by bernardsilenou** [comment]
> @bernardsilenou if we restrict to a particular disease when creating this user, does it restrict the statistics menu to only data from the selected disease?

NO, statistics does not real depends on the user role. That is why a surveillance supervisor can quarry and get the total number of cases in the country. We need to test this to be 100% sure

**Comment by bernardsilenou** [comment]
> @bernardsilenou No, national users also have access to database export.
> 
> * right to manipulate the statistics menu as it is
> * They should only see the statistics and dashboard menu

If this is the only right that the user need to have, then they are not part of the surveillance unit, right?
Can the admin not just export the data and give the user?
How urgent is this needed? can we keep it till when we have the user right management integrated?

**Comment by Chinedar** [comment]
> 
> 
> > @bernardsilenou No, national users also have access to database export.
> > 
> > * right to manipulate the statistics menu as it is
> > * They should only see the statistics and dashboard menu
> 
> If this is the only right that the user need to have, then they are not part of the surveillance unit, right?
> Can the admin not just export the data and give the user?
> How urgent is this needed? can we keep it till when we have the user right management integrated?

@bernardsilenou They are definitely not part of the surveillance team. This is more of a political request to gain more buy-in from partners and stakeholders who do not need to see all case records, but may be allowed to generate basic analytics. 
When is the user right management likely to be integrated?

**Comment by bernardsilenou** [comment]
OK, I understand better now. We have a roadmap meeting next week or the other week and we will then be able to decide when to begin implementing the user right management. Lets followup on this next week please. Thanks
